react/lib/components/Widget/Widget.tsx

```diff
@@ -629,15 +629,17 @@ export const Widget: React.FunctionComponent<WidgetProps> = props => {
           py={1}
           textAlign="center"
         >
-          {errorMsg ? (
-            <Typography className={classes.error}>
-              {errorMsg}
-            </Typography>
-          ) : (
-            <Typography className={classes.text}>
-              {loading ? 'Loading...' : success ? successText : text}
-            </Typography>
-          )}
+          <Typography className={errorMsg ? classes.error : classes.text}>
+          {errorMsg 
+            ? errorMsg 
+            : disabled 
```

Comment:
maybe some parenthesis and indentation here to make it more readable? The change makes the begginer more readable, but the part in the middle is quite hard to follow